lib/github/buckets.rb

```diff
@@ -23,11 +27,11 @@ def all
     end
 
     def all_members
-      members.concat(contractors)
+      members.to_a + contractors.to_a
```

Comment:
You can just do `members + contractors`, these are already arrays.

Reply:
@dblock They arent arrays, they are Hashs. Atleast in [`contributors.rb`](https://github.com/opensearch-project/project-tools/blob/main/lib/github/contributors.rb#L41) they are. When i run pr stats, it fails unless they are explicitly converted to arrays

Reply:
Oh but that is a problem, you get back an array. We need to merge hashes and get back a hash, don't we?

Write a test for this, and you'll see what works.